paasta_tools/contrib/ide_helper.py

```diff
@@ -144,6 +163,67 @@ def install_vscode_support() -> None:
                     "PYTHONUNBUFFERED": "1",
                 },
             },
+            {
+                "name": "paasta API playground",
+                "cwd": "${workspaceFolder}",
+                "python": "${workspaceFolder}/.tox/py37-linux/bin/python",
+                "type": "python",
+                "request": "launch",
+                "module": "paasta_tools.run-paasta-api-playground",
+                "justMyCode": False,
+                "env": {
+                    "KUBECONFIG": "./k8s_itests/kubeconfig",
+                    "PAASTA_SYSTEM_CONFIG_DIR": "./etc_paasta_playground/",
+                    "PAASTA_TEST_CLUSTER": "kind-emanelsabban-k8s-test",
+                    "PYDEVD_USE_CYTHON": "NO",
+                    "PYTHONUNBUFFERED": "1",
+                    "PAASTA_API_SOA_DIR": "./k8s_itests/deployments/paasta/fake_soa_config",
+                },
+            },
+            {
+                "name": "Run setup k8s job",
```

Comment:
suggestion: might be nice to add `in playground` or something to the names of these